### PR TITLE
Added parameters needed by Raul

### PR DESCRIPTION
**Pull request by jpsandovaln**

### CODE REVIEW

File: frontend/src/app/components/users/birthday/users.birthday.controller.js [hunk 1]
```diff
@@ -1,29 +1,40 @@
...
+ * Controller for Birthday
+ */
 export default class BirthdayController {
     constructor($interval, $http, $routeParams) {
         'ngInject';
         this.$interval = $interval;
         this.$http = $http;
         this.$routeParams = $routeParams;
         this.employee_id = this.$routeParams.id;
+        this.employee_firstName = this.$routeParams.firstName;
```
Comment (camba95) [inline frontend/src/app/components/users/birthday/users.birthday.controller.js]: Please use camel case e.g. longVariableName

File: frontend/src/app/components/users/birthday/users.birthday.controller.js [hunk 2]
```diff
@@ -1,29 +1,40 @@
...
+        this.employee_lastName = this.$routeParams.lastName;
+        this.employee_photoUrl = this.$routeParams.photoUrl;
         this.employees = [];
         this.baseUri = `http://${config.serverHost}:${config.serverPort}`;
         this.employeeRoute = `${this.baseUri}/crud/employees`;
         if (this.employee_id === undefined || this.employee_id === null) {
-            this.urlBirthday = `${this.employeeRoute}/birthdaylist`;
+            if (this.employee_firstName === undefined && this.employee_lastName === undefined && this.employee_photoUrl  === undefined) {
```
Comment (camba95) [inline frontend/src/app/components/users/birthday/users.birthday.controller.js]: Use falsy conditional or comparison
Not 'if (varible === undefined)'
Instead 'if (!variable)'

**Review by camba95 — CHANGES_REQUESTED** [review]

File: frontend/src/app/components/users/birthday/users.birthday.controller.js (hunk 2 above)
Comment (jpsandovaln) [reply frontend/src/app/components/users/birthday/users.birthday.controller.js]: I fixed your observations, please could you review again?

File: frontend/src/app/components/users/birthday/users.birthday.controller.js [hunk 3]
```diff
@@ -44,19 +44,19 @@ export default class BirthdayController {
...
-                    this.employee_name = this.employee.firstName + ' ' + this.employee.lastName;
+                    this.employeeName = this.employee.firstName + ' ' + this.employee.lastName;
                     this.employeeImg = this.employee.image;
                 } else {
                     this.employeeImg = this.defaultImg;
                 }
             }
         }, (error) => {
-            console.log(error, 'can not get data.');
+            console.er(error, 'can not get data.');
```
Comment (camba95) [inline frontend/src/app/components/users/birthday/users.birthday.controller.js]: `console.er` does not exist, this will raise an error

**Review by camba95 — CHANGES_REQUESTED** [review]

File: frontend/src/app/components/users/birthday/users.birthday.controller.js (hunk 3 above)
Comment (jpsandovaln) [reply frontend/src/app/components/users/birthday/users.birthday.controller.js]: fixed

**Review by camba95 — APPROVED** [review]

**Pull request merged**